File: src/net/merayen/elastic/backend/architectures/local/nodes/group_1/LProcessor.java
package net.merayen.elastic.backend.architectures.local.nodes.group_1;

import net.merayen.elastic.backend.architectures.local.LocalProcessor;
import net.merayen.elastic.util.Postmaster.Message;

public class LProcessor extends LocalProcessor {
	int session_id = -1;

	@Override
	protected void onInit() {}

	@Override
	protected void onPrepare() {}

	@Override
	protected void onProcess() {
		if(session_id == -1)
			session_id = spawnSession(0);
	}

	@Override
	protected void onMessage(Message message) {}

	@Override
	protected void onDestroy() {}

}

File: src/net/merayen/elastic/backend/architectures/local/nodes/eq_1/LProcessor.kt
package net.merayen.elastic.backend.architectures.local.nodes.eq_1

import net.merayen.elastic.backend.architectures.local.LocalProcessor
import net.merayen.elastic.util.Postmaster

class LProcessor : LocalProcessor() {
	override fun onInit() {}
	override fun onPrepare() {}
	override fun onProcess() {}
	override fun onMessage(message: Postmaster.Message?) {}
	override fun onDestroy() {}
}